modules/social_features/social_core/social_core.module

```diff
@@ -522,6 +524,35 @@ function social_core_filter_process_format($element) {
   return $element;
 }
 
+/**
+ * Hide blocks on extendable visibility path.
+ *
+ * Implements hook_block_access().
+ */
+function social_core_block_access(Block $block, $operation, AccountInterface $account) {
+  $blocks = &drupal_static(__FUNCTION__);
+  $blocks = [];
+
+  // Allow other modules to change the block visibility.
+  \Drupal::moduleHandler()->alter('social_core_block_visibility_path', $blocks);
+
+  if ($operation === 'view' && array_key_exists($block->getPluginId(), $blocks)) {
```

Comment:
Instead of `array_key_exists` we can use `isset`. It's [slightly faster](https://stackoverflow.com/a/9522522/576060) and provides [a bit more robustness](https://stackoverflow.com/a/3210982/576060) about incorrectly specified values.

```suggestion
  if ($operation === 'view' && isset($blocks[$block->getPluginId()])) {
```